great_expectations/compatibility/pyspark.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.not_imported import NotImported
+
+SPARK_NOT_IMPORTED = NotImported(
+    "pyspark is not installed, please 'pip install pyspark'"
+)
+
+try:
+    import pyspark  # noqa: TID251
+except (ImportError, AttributeError):
```

Comment:
No need for AttributeError here. Only if we are accessing something within a package (like `from pyspark.sql import functions`). (This is applicable to the other files as well)

Reply:
@anthonyburdi I agree -- let me see how far I can go with removing `AttributeError` (it was extra precaution).  Thanks.

Reply:
@anthonyburdi This was done -- I deleted `AttributeError` wherever it was superfluous.  Thanks.